okio/jvm/src/main/java/okio/-Platform.kt

```diff
@@ -34,15 +32,9 @@ internal actual fun arraycopy(
   System.arraycopy(src, srcPos, dest, destPos, length)
 }
 
-internal actual fun hashCode(a: ByteArray): Int = Arrays.hashCode(a)
-
 internal actual fun ByteArray.toUtf8String(): String = String(this, Charsets.UTF_8)
 
-internal actual fun CharArray.createString(): String = String(this)
-
 internal actual fun String.asUtf8ToByteArray(): ByteArray = toByteArray(Charsets.UTF_8)
 
 // TODO remove if https://youtrack.jetbrains.com/issue/KT-20641 provides a better solution
```

Comment:
Should this comment be removed?

Reply:
Yes, it should be removed. Looks like I removed it from JS and Native; must have missed this one.